<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginForm</class>
 <widget class="QDialog" name="LoginForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParfÃ¼m Koleksiyonu - GiriÅŸ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #1e293b;
}

QLabel#label_baslik {
    color: #f3f4f6;
    font-size: 24px;
    font-weight: bold;
}

QLabel#label_alt_baslik {
    color: #cbd5e1;
    font-size: 14px;
}

QLabel {
    color: #cbd5e1;
    font-size: 14px;
    font-weight: bold;
}

QLineEdit {
    padding: 10px;
    border: 1px solid #475569;
    border-radius: 8px;
    background-color: #334155;
    color: #f3f4f6;
    font-size: 14px;
}

QLineEdit:focus {
    border: 2px solid #3b82f6;
}

QPushButton {
    background-color: #3b82f6;
    color: white;
    border-radius: 8px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #2563eb;
}

QPushButton#btn_kayit {
    background-color: #10b981;
}

QPushButton#btn_kayit:hover {
    background-color: #059669;
}

QLabel#lbl_hata {
    color: #ef4444;
    font-weight: bold;
    min-height: 35px;
    background-color: rgba(239, 68, 68, 0.1);
    border-radius: 5px;
    padding: 5px;
    margin-top: 5px;
    margin-bottom: 5px;
    line-height: 25px;
}

#icon_container {
    background-color: #3b82f6;
    border-radius: 10px;
}

#welcome_label {
    color: #f3f4f6;
    font-size: 20px;
    font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="icon_container">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="icon_label">
          <property name="text">
           <string>ðŸ‘¤</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size: 36px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
       </property>
       <item>
        <widget class="QLabel" name="label_baslik">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
            color: #fff;
            font-size: 22px;
            font-weight: bold;
            background-color: rgba(10, 10, 10, 0.92);
            border: 3px solid #fff;
            border-radius: 14px;
            padding: 6px 12px;
            letter-spacing: 1px;
          </string>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred"/>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>ParfÃ¼m Koleksiyonu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_alt_baslik">
         <property name="text">
          <string>KiÅŸisel koleksiyon uygulamanÄ±za giriÅŸ yapÄ±n</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="login_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#login_frame {
    background-color: #273549;
    border-radius: 12px;
    border: 1px solid #475569;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>25</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>25</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="welcome_label">
        <property name="styleSheet">
         <string notr="true">color: #f3f4f6;
font-size: 20px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string>HoÅŸ Geldiniz</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbl_hata">
        <property name="styleSheet">
         <string notr="true">color: #ef4444;
font-weight: bold;
min-height: 35px;
background-color: rgba(239, 68, 68, 0.1);
border-radius: 5px;
padding: 5px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="username_layout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_kullanici">
          <property name="styleSheet">
           <string notr="true">color: #cbd5e1;</string>
          </property>
          <property name="text">
           <string>KullanÄ±cÄ± AdÄ±</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_kullanici_adi">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>KullanÄ±cÄ± adÄ±nÄ±zÄ± girin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="password_layout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_sifre">
          <property name="styleSheet">
           <string notr="true">color: #cbd5e1;</string>
          </property>
          <property name="text">
           <string>Åžifre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txt_sifre">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Åžifrenizi girin</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_giris">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>GiriÅŸ Yap</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_kayit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Yeni Hesap OluÅŸtur</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 